storage_subnet/dht/storage.py

```diff
@@ -0,0 +1,137 @@
+from typing import Union
+
+from kademlia.storage import IStorage
+
+from storage_subnet.dht.chunk_dht import ChunkDHTValue
+from storage_subnet.dht.piece_dht import PieceDHTValue
+from storage_subnet.dht.tracker_dht import TrackerDHTValue
+from storage_subnet.validator import db
+from storage_subnet.validator.db import ChunkEntry, PieceEntry, TrackerEntry
+
+DHTValue = Union[ChunkDHTValue, PieceDHTValue, TrackerDHTValue]
+
+
+class SqliteStorageDHT(IStorage):
+    async def __init__(self):
+        self.db_dir = db.DB_DIR
+        self.db_conn = await db.get_db_connection(self.db_dir)
```

Comment:
the `db_dir` has to be an argument, as the validator defines where the db is located (see `config.py`).